src/stylus/components/_overlay.styl

```diff
@@ -7,6 +7,12 @@
   right: 0
   bottom: 0
   pointer-events: none
+  // The overlay has a dynamically set
+  // z-index, we want the transition
+  // timing to affect its changing
+  // https://github.com/vuetifyjs/vuetify/issues/2146
+  transition: .5s $transition.swing
```

Comment:
0.5s here and 150ms below is calculated by trial and error method or are they related to some other props? Maybe they could be made as a consts and resused

Reply:
Trial and error